2025-03-09 23:25:03,719 INFO    StreamThr :21688 [internal.py:wandb_internal():86] W&B internal server running at pid: 21688, started at: 2025-03-09 23:25:03.718565
2025-03-09 23:25:03,721 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status
2025-03-09 23:25:03,749 INFO    WriterThread:21688 [datastore.py:open_for_write():85] open: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\run-qebtyw5e.wandb
2025-03-09 23:25:03,751 DEBUG   SenderThread:21688 [sender.py:send():382] send: header
2025-03-09 23:25:03,807 DEBUG   SenderThread:21688 [sender.py:send():382] send: run
2025-03-09 23:25:04,436 INFO    SenderThread:21688 [dir_watcher.py:__init__():211] watching files in: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files
2025-03-09 23:25:04,436 INFO    SenderThread:21688 [sender.py:_start_run_threads():1136] run started: qebtyw5e with start time 1741542903.718565
2025-03-09 23:25:04,449 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: check_version
2025-03-09 23:25:04,450 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: check_version
2025-03-09 23:25:06,287 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: run_start
2025-03-09 23:25:07,283 DEBUG   HandlerThread:21688 [system_info.py:__init__():32] System info init
2025-03-09 23:25:07,283 DEBUG   HandlerThread:21688 [system_info.py:__init__():47] System info init done
2025-03-09 23:25:07,283 INFO    HandlerThread:21688 [system_monitor.py:start():194] Starting system monitor
2025-03-09 23:25:07,283 INFO    SystemMonitor:21688 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-03-09 23:25:07,283 INFO    HandlerThread:21688 [system_monitor.py:probe():214] Collecting system info
2025-03-09 23:25:07,287 INFO    SystemMonitor:21688 [interfaces.py:start():190] Started cpu monitoring
2025-03-09 23:25:07,288 INFO    SystemMonitor:21688 [interfaces.py:start():190] Started disk monitoring
2025-03-09 23:25:07,289 INFO    SystemMonitor:21688 [interfaces.py:start():190] Started gpu monitoring
2025-03-09 23:25:07,297 INFO    SystemMonitor:21688 [interfaces.py:start():190] Started memory monitoring
2025-03-09 23:25:07,313 INFO    SystemMonitor:21688 [interfaces.py:start():190] Started network monitoring
2025-03-09 23:25:07,317 DEBUG   HandlerThread:21688 [system_info.py:probe():196] Probing system
2025-03-09 23:25:07,318 DEBUG   HandlerThread:21688 [system_info.py:_probe_git():181] Probing git
2025-03-09 23:25:07,407 DEBUG   HandlerThread:21688 [system_info.py:_probe_git():189] Probing git done
2025-03-09 23:25:07,407 DEBUG   HandlerThread:21688 [system_info.py:probe():244] Probing system done
2025-03-09 23:25:07,407 DEBUG   HandlerThread:21688 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.26100-SP0', 'python': '3.9.5', 'heartbeatAt': '2025-03-09T17:55:07.317730', 'startedAt': '2025-03-09T17:55:03.703600', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'git@github.com-work:saimanikumar-da24m016/da6401_assignment1.git', 'commit': '3ddbfd47c7127abba95926229ee5c61c7e5e5eb9'}, 'email': 'saimanikumar67@gmail.com', 'root': 'C:/Users/D. Sai Mani Kumar/Downloads/DL/da6401_assignment1', 'host': 'LAPTOP-VRI27UL2', 'username': 'D. Sai Mani Kumar', 'executable': 'c:\\Users\\D. Sai Mani Kumar\\AppData\\Local\\Programs\\Python\\Python39\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 2400.0, 'min': 0.0, 'max': 2400.0}, 'cpu_freq_per_core': [{'current': 2400.0, 'min': 0.0, 'max': 2400.0}], 'disk': {'/': {'total': 375.82519149780273, 'used': 261.9075584411621}}, 'gpu': 'NVIDIA GeForce GTX 1650 Ti', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GTX 1650 Ti', 'memory_total': 4294967296}], 'memory': {'total': 7.840869903564453}}
2025-03-09 23:25:07,407 INFO    HandlerThread:21688 [system_monitor.py:probe():224] Finished collecting system info
2025-03-09 23:25:07,407 INFO    HandlerThread:21688 [system_monitor.py:probe():227] Publishing system info
2025-03-09 23:25:07,407 DEBUG   HandlerThread:21688 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2025-03-09 23:25:07,409 DEBUG   HandlerThread:21688 [system_info.py:_save_pip():68] Saving pip packages done
2025-03-09 23:25:07,410 INFO    HandlerThread:21688 [system_monitor.py:probe():229] Finished publishing system info
2025-03-09 23:25:07,423 DEBUG   SenderThread:21688 [sender.py:send():382] send: files
2025-03-09 23:25:07,423 INFO    SenderThread:21688 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2025-03-09 23:25:07,450 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\requirements.txt
2025-03-09 23:25:07,451 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\wandb-metadata.json
2025-03-09 23:25:07,451 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:25:07,452 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:25:07,455 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:25:07,866 DEBUG   SenderThread:21688 [sender.py:send():382] send: telemetry
2025-03-09 23:25:07,866 DEBUG   SenderThread:21688 [sender.py:send():382] send: telemetry
2025-03-09 23:25:07,866 DEBUG   SenderThread:21688 [sender.py:send():382] send: run
2025-03-09 23:25:08,171 INFO    SenderThread:21688 [sender.py:send_run():1010] updated run: qebtyw5e
2025-03-09 23:25:08,459 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\config.yaml
2025-03-09 23:25:08,461 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:25:08,960 INFO    wandb-upload_0:21688 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmptnk4gvsiwandb\1bkkiy3h-wandb-metadata.json
2025-03-09 23:25:09,180 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:09,364 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:09,365 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:10,465 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:25:11,775 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:11,775 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:14,176 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:14,178 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:14,224 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:16,580 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:16,580 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:18,980 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:18,981 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:19,250 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:21,386 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:21,386 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:22,457 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:25:22,458 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:25:22,472 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:25:23,784 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:23,784 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:24,789 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:24,790 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 23:25:24,791 DEBUG   SenderThread:21688 [sender.py:send():382] send: history
2025-03-09 23:25:24,791 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: summary_record
2025-03-09 23:25:24,795 INFO    SenderThread:21688 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 23:25:25,631 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\wandb-summary.json
2025-03-09 23:25:26,179 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:26,179 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:26,636 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:25:28,573 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:28,573 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:29,847 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:30,978 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:30,979 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:33,388 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:33,388 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:34,889 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:35,782 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\config.yaml
2025-03-09 23:25:35,791 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:35,792 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:37,471 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:25:37,472 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:25:37,487 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:25:38,190 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:38,191 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:39,834 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 23:25:39,836 DEBUG   SenderThread:21688 [sender.py:send():382] send: history
2025-03-09 23:25:39,836 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: summary_record
2025-03-09 23:25:39,839 INFO    SenderThread:21688 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 23:25:39,839 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\wandb-summary.json
2025-03-09 23:25:40,596 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:40,596 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:40,848 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:40,849 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:25:45,552 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:45,552 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:45,938 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:47,973 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:47,973 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:50,371 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:50,371 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:51,003 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:52,481 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:25:52,482 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:25:52,497 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:25:52,777 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:52,778 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:55,185 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:55,186 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:55,747 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 23:25:55,748 DEBUG   SenderThread:21688 [sender.py:send():382] send: history
2025-03-09 23:25:55,748 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: summary_record
2025-03-09 23:25:55,751 INFO    SenderThread:21688 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 23:25:56,105 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\wandb-summary.json
2025-03-09 23:25:56,755 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:25:57,117 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:25:57,579 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:57,580 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:59,985 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:25:59,985 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:01,799 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:02,378 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:02,378 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:04,793 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:04,794 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:06,843 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:07,205 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:07,205 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:07,317 DEBUG   SystemMonitor:21688 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-03-09 23:26:07,319 DEBUG   SenderThread:21688 [sender.py:send():382] send: stats
2025-03-09 23:26:07,490 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:26:07,490 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:26:07,491 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:26:09,606 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:09,606 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:11,870 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:12,006 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:12,007 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:12,191 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 23:26:12,194 DEBUG   SenderThread:21688 [sender.py:send():382] send: history
2025-03-09 23:26:12,194 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: summary_record
2025-03-09 23:26:12,199 INFO    SenderThread:21688 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 23:26:12,371 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\wandb-summary.json
2025-03-09 23:26:14,406 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:14,406 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:15,423 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:26:16,822 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:16,823 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:17,248 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:21,765 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:21,765 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:22,287 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:22,505 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:26:22,505 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:26:22,506 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:26:24,159 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:24,160 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:26,556 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:26,557 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:27,860 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:28,966 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:28,966 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:29,754 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 23:26:29,755 DEBUG   SenderThread:21688 [sender.py:send():382] send: history
2025-03-09 23:26:29,755 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: summary_record
2025-03-09 23:26:29,756 INFO    SenderThread:21688 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 23:26:30,692 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:26:30,694 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\wandb-summary.json
2025-03-09 23:26:31,359 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:31,360 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:33,760 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:33,760 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:33,790 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:36,165 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:36,165 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:37,330 DEBUG   SenderThread:21688 [sender.py:send():382] send: stats
2025-03-09 23:26:37,520 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:26:37,520 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:26:37,521 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:26:38,574 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:38,575 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:38,856 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:40,987 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:40,987 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:43,383 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:43,384 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:43,917 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:45,789 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:45,789 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:45,967 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 23:26:45,968 DEBUG   SenderThread:21688 [sender.py:send():382] send: history
2025-03-09 23:26:45,969 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: summary_record
2025-03-09 23:26:45,971 INFO    SenderThread:21688 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 23:26:46,965 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:26:46,966 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\wandb-summary.json
2025-03-09 23:26:48,199 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:48,200 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:48,999 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:50,606 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:50,606 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:52,537 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:26:52,537 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:26:52,538 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:26:52,998 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:52,999 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:54,871 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:26:57,898 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:57,899 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:26:59,906 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:27:00,286 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:00,286 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:02,693 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:02,694 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:03,090 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 23:27:03,092 DEBUG   SenderThread:21688 [sender.py:send():382] send: history
2025-03-09 23:27:03,092 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: summary_record
2025-03-09 23:27:03,094 INFO    SenderThread:21688 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 23:27:03,255 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\wandb-summary.json
2025-03-09 23:27:05,107 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:27:05,108 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:05,108 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:05,288 INFO    Thread-16 :21688 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_232503-qebtyw5e\files\output.log
2025-03-09 23:27:07,335 DEBUG   SenderThread:21688 [sender.py:send():382] send: stats
2025-03-09 23:27:07,507 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:07,507 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:07,555 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:27:07,556 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:27:07,556 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:27:09,899 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:09,899 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:10,913 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:27:12,317 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:12,318 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:14,733 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:14,734 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:15,967 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:27:17,235 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:17,236 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:19,676 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:19,677 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:21,023 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:27:22,087 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:22,089 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:22,561 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 23:27:22,583 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 23:27:22,584 DEBUG   SenderThread:21688 [sender.py:send_request():409] send_request: stop_status
2025-03-09 23:27:24,507 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:24,508 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:26,926 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:26,926 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:26,942 DEBUG   HandlerThread:21688 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 23:27:29,344 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 23:27:29,345 ERROR   gpu       :21688 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
